### ISSUE TITLE
TextInputLayout Accessibility test fail with “View is missing speakable text needed for a screen reader”

### ISSUE THREAD

**Issue by fartem** [material-components/material-components-android#1040]
During testing of Accessibility, tests on EditText in TextInputLayout fail with "View is missing speakable text needed for a screen reader". A similar issue is described here, but the proposed solutions with XML parameters do not help. Tested on library versions 1.1.0 and 1.2.0-alpha04.

Layout part:

```xml
<com.google.android.material.textfield.TextInputLayout
    android:id="@+id/login_email_address"
    android:layout_width="0dp"
    android:layout_height="wrap_content"
    android:layout_marginTop="@dimen/padding_l"
    android:layout_marginLeft="@dimen/padding_l"
    android:layout_marginStart="@dimen/padding_l"
    android:layout_marginRight="@dimen/padding_l"
    android:layout_marginEnd="@dimen/padding_l"
    android:labelFor="@id/login_email_address_edit_text"
    android:accessibilityHeading="true"
    android:contentDescription="@string/login_email_address"
    app:layout_constraintTop_toBottomOf="@id/login_error_message"
    app:layout_constraintLeft_toLeftOf="parent"
    app:layout_constraintRight_toRightOf="parent"
    app:errorTextAppearance="@style/ErrorText"
    app:errorEnabled="true">

    <com.google.android.material.textfield.TextInputEditText
        android:id="@+id/login_email_address_edit_text"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:minHeight="@dimen/edit_text_min_height"
        android:inputType="textEmailAddress"
        android:hint="@string/login_email_address" />

</com.google.android.material.textfield.TextInputLayout>
```

**Comment by leticiarossi** [comment]
Hi, `android:hint` should be set on the `TextInputLayout` instead of on the `TextInputEditText`.
Does changing that solve your issue?

**Comment by fartem** [comment]
Nope.

**Comment by drchen** [comment]
Is this still an issue? I don't see similar situations happening with our own accessibility tests.

**Comment by fartem** [comment]
@drchen, not sure about this issue right now. Project when I faced with this issue not maintained right now. If you need, I can try to test this issue and tell about it on new version of the library.

**Comment by drchen** [comment]
Thanks @fartem. I'm going to close the issue for now. Feel free to reopen it if you are still seeing this happening! : )
